CofeShop/Program.cs

```diff
@@ -1,29 +1,64 @@
 ﻿using System;
 using System.IO;
+using Microsoft.Extensions.Configuration;
+using Microsoft.EntityFrameworkCore;
 using System.Threading.Tasks;
-using Newtonsoft.Json.Converters;
-using Newtonsoft.Json;
 using CofeShop.Models;
 using CofeShop.Controllers;
+using System.Linq;
 using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore.Infrastructure;
 
 namespace CofeShop
 {
-    class Program
+    public class Program
     {
-        static string path = (Directory.GetCurrentDirectory() + "Source.json");
+        
 
-        static void Main(string[] args)
+        public static void Main(string[] args)
         {
-            
-            Console.WriteLine(Directory.GetCurrentDirectory());
-            SourceController.LoadJsonSource();
-            SourceController.ReadJsonSource();
-           //SourceController.ReadJsonSource();
-           //List <Goods> items = Goods.ReadFromSource(path + "\\Models\\Source.json");
-           //Console.WriteLine(items.Count);
-        }
+            int choice = 2;
+           
 
-        
+            GoodsController g = new GoodsController();
```

Comment:
I don't really understand why do you need this, but ok, let it be. 

But i strongly recommend you to use your controller's methods as app entry points. 
It can be done by REST requests. 
You can test it without UI by Postman.